utils/post_utils.jsx

```diff
@@ -216,3 +216,54 @@ export function isErrorInvalidSlashCommand(error) {
 
     return false;
 }
+
+function parseTable(html) {
```

Comment:
Perhaps we could move all of these functions into their own file named something like `utils/paste.js`? They're self-contained, so it might be nice to keep them on their own.